docs/source/release_notes/release-notes.rst

```diff
@@ -82,6 +82,16 @@ develop
            Instead, the exception is logged once only, when we run out of retries.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2432>`__)
 
+    *    - |new|
+         - AtlasDB clients now emit metrics that track the immutable timestamp, unreadable timestamp as well as current timestamp.
+           These metrics should help in performing diagnosis of issues concerning Sweep and/or the lock service.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/ABCD>`__)
```

Comment:
reminder: PR number (I wouldn't put this, but I found a link on develop recently with `pull/ABCD`)!

Reply:
Ack. Also, the `ABCD` is probably me - I'm not sure what placeholders others write, but I know that I often use `ABCD` (and sometimes `EFGH` for multiple-entry changes) 😓